<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>440</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="4">
     <widget class="QCheckBox" name="checkBox_3">
      <property name="text">
       <string>Subrayado</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Tipografía</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QFontComboBox" name="fontComboBox"/>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="colorMarco">
      <property name="text">
       <string>Color Marco</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Epigrafe elegido</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="colorFondo">
      <property name="text">
       <string>Color Fondo</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="colorLetra">
      <property name="text">
       <string>Color Letra</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>:/Img/folks.PNG</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>:/Img/flecha.png</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Imagen</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Imagen elegida</string>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QLineEdit" name="lineEdit_6"/>
    </item>
    <item row="2" column="3">
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string>Italic</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="9">
     <widget class="SAACWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Ancho Marco</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Tamaño Letra</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QCheckBox" name="checkMayuscula">
      <property name="text">
       <string>Minuscula</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QLineEdit" name="lineEdit_5"/>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Epígrafe</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Negrita</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="8" column="6">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="plainText">
       <string>default
text</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SAACWidget</class>
   <extends>QWidget</extends>
   <header>..\saacwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
